FirebaseCore/Tests/Unit/Swift/FirebaseAppTests.swift

```diff
@@ -77,120 +86,104 @@ class FirebaseAppTests: XCTestCase {
-    options.clientID = testOptions.clientID
+    let options = FirebaseOptions(googleAppID: Constants.Options.googleAppID,
+                                  gcmSenderID: Constants.Options.gcmSenderID)
+    options.clientID = Constants.Options.clientID
     let configurationAttempt = {
       try ExceptionCatcher.catchException {
         FirebaseApp.configure(options: options)
       }
     }
     XCTAssertNoThrow(try configurationAttempt())
 
-    // TODO: Use XCTUnwrap after dropping support for Xcode 10
-    guard let app = FirebaseApp.app() else {
-      return XCTFail("Failed to unwrap default app")
-    }
-    XCTAssertEqual(app.name, testApp.firebaseDefaultAppName)
-    XCTAssertEqual(app.options.googleAppID, testOptions.googleAppID)
-    XCTAssertEqual(app.options.gcmSenderID, testOptions.gcmSenderID)
-    XCTAssertEqual(app.options.clientID, testOptions.clientID)
+    let app = try XCTUnwrap(FirebaseApp.app(), "Failed to unwrap default app")
+    XCTAssertEqual(app.name, Constants.App.defaultName)
+    XCTAssertEqual(app.options.googleAppID, Constants.Options.googleAppID)
+    XCTAssertEqual(app.options.gcmSenderID, Constants.Options.gcmSenderID)
+    XCTAssertEqual(app.options.clientID, Constants.Options.clientID)
     XCTAssertTrue(FirebaseApp.allApps?.count == 1)
 
-    waitForExpectations()
+    waitForExpectations(timeout: 1)
   }
 
-  func testConfigureWithNameAndOptions() {
-    let expectedUserInfo = expectedUserInfoForApp(named: testApp.testAppName1,
-                                                  isDefaultApp: false)
-    expectAppConfigurationNotification(with: expectedUserInfo)
+  func testConfigureWithNameAndOptions() throws {
+    expectAppConfigurationNotification(appName: Constants.testAppName1, isDefaultApp: false)
 
-    let options = FirebaseOptions(googleAppID: testOptions.googleAppID,
-                                  gcmSenderID: testOptions.gcmSenderID)
-    options.clientID = testOptions.clientID
+    let options = FirebaseOptions(googleAppID: Constants.Options.googleAppID,
+                                  gcmSenderID: Constants.Options.gcmSenderID)
+    options.clientID = Constants.Options.clientID
 
     let configurationAttempt = {
       try ExceptionCatcher.catchException {
-        FirebaseApp.configure(name: self.testApp.testAppName1, options: options)
+        FirebaseApp.configure(name: Constants.testAppName1, options: options)
       }
     }
     XCTAssertNoThrow(try configurationAttempt())
 
-    guard let app = FirebaseApp.app(name: testApp.testAppName1) else {
-      return XCTFail("Failed to unwrap default app")
-    }
-    XCTAssertEqual(app.name, testApp.testAppName1)
-    XCTAssertEqual(app.options.googleAppID, testOptions.googleAppID)
-    XCTAssertEqual(app.options.gcmSenderID, testOptions.gcmSenderID)
-    XCTAssertEqual(app.options.clientID, testOptions.clientID)
+    let app = try XCTUnwrap(
+      FirebaseApp.app(name: Constants.testAppName1),
+      "Failed to unwrap default app"
```

Comment:
Nit that I missed previously: this isn't the default app, it should be `Failed to unwrap custom named app.`

Reply:
done